explorer/src/lib/components/transaction-details/TransactionDetails.svelte

```diff
@@ -33,8 +33,11 @@
   /** @type {String | null} */
   export let payload;
 
+  /** @type {MarketData}*/
```

Comment:
should this be `MarketData | null`?

Reply:
It should normally, but then I'd have to add a `ts-ignore` to line 142 and 152

Reply:
Annoying.
We'll think about how to handle those cases: maybe add a normal comment in the meanwhile to say that it can be `null`?